content/en/docs/concepts/scheduling-eviction/pod-priority-preemption.md

```diff
@@ -187,6 +187,8 @@ scheduler will continue and try to schedule other lower priority Pods.
 
 ## Preemption
 
+{{< feature-state for_k8s_version="v1.14" state="stable" >}}
```

Comment:
Did you mean to list this under the heading `## PriorityClass`?

Reply:
(that's where I'd put it, and **drop** the feature state from the state of the page)